_rules/html-page-lang-b5c3f8.md

```diff
@@ -29,11 +29,11 @@ This rule applies to any [document element](https://dom.spec.whatwg.org/#documen
 
 ## Expectation
 
-Each test target has a `lang` attribute that is neither empty ("") nor only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace).
+Each test target has a `lang` attribute that is neither empty (`""`) nor only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace).
 
 ## Assumptions
 
-This rule assumes that the presence of a `lang` attribute is being used to comply to WCAG. This rule doesn't test if the attribute is needed to comply to WCAG.
+This rule assumes that the presence of a `lang` attribute is being used to comply to [WCAG][]. This rule doesn't test if the attribute is needed to comply to [WCAG][].
```

Comment:
This seems unnecessary. Nobody reading this document is going to be confused about what WCAG is or where to find it.